packages/modals/src/styled/StyledTooltipModalClose.ts

```diff
@@ -13,13 +13,11 @@ const COMPONENT_ID = 'modals.tooltip_modal.close';
 
 export const StyledTooltipModalClose = styled(StyledClose).attrs({
   'data-garden-id': COMPONENT_ID,
-  'data-garden-version': PACKAGE_VERSION
+  'data-garden-version': PACKAGE_VERSION,
+  size: 'small'
```

Comment:
[nit] the general Garden preference is to locate these props on the element component – where they provide somewhat better prominence. Styled components that show otherwise are anti-patterns.

Reply:
Is that what you expected? 

https://github.com/zendeskgarden/react-components/pull/1840/commits/e334bef5cabb36712f54099e00c429b7c50fe9b9

Reply:
Yes :+1:. Matches @geotrev's treatment in #1842